### DISCUSSION TITLE
Illinois GitHub Enterprise License

### DISCUSSION THREAD

**Discussion by gsrohde** [biocro/biocro#124] [General]
This came through on the the Technology Services Privacy & Cybersecurity mailing list.  I haven't looked into this in depth, but it may be something we want to consider.

---
Resources for Using GitHub Enterprise
 
 	
Strong version control of custom code and configurations is a key part of assurance and agility for campus systems. The Illinois GitHub Enterprise license is a good option for achieving this agility and assurance.

The shared GitHub service provides a platform for university collaboration, reduces the cost for existing users of GitHub, makes GitHub accessible to new users, provides a way to promote U of I System open source projects and improves security by providing a shared GitHub platform that follows a standard set of default security policies.

As a licensed service available to all of campus, GitHub usually has the largest number of supporting IT professional community articles, and knowledge base job-aid recipes, among the available solutions for version control. 

Security events from GitHub Enterprise are logged to the IPC Splunk instance, enhancing incident response capabilities, should a Cybersecurity event related to source control occur.

An overview of the service is available at https://web.uillinois.edu/github, and campus-specific security and technical recipes for campus GitHub users are available at https://answers.illinois.edu/search.php?q=github and https://uillinois-community.github.io/

**Comment by justinmcgrath** [comment]
Do you know what this provides that we don't have already?
